.github/workflows/deploy-pages.yml

```diff
@@ -27,7 +27,7 @@ jobs:
             clang-runtime: '19'
             cling: Off
             micromamba_shell_init: bash
-            emsdk_ver: "3.1.45"
+            emsdk_ver: "3.1.73"
```

Comment:
This can completely be removed once the above comment is addressed

Reply:
Similarly for docs and the main.yml. We wouldn't need to store this in a variable !